px-dropdown-content.html

```diff
@@ -80,7 +80,7 @@
                 <px-icon icon="px-utl:check" class="select-icon u-pr--"></px-icon>
               </template>
               <template is="dom-if" if="{{item.icon}}">
-                <px-icon icon="{{item.icon}}" class="item-icon u-mr-" style$="color:{{item.color}};stroke:{{item.color}}"></px-icon>
```

Comment:
I don't think that this classname should have been removed.

Reply:
Added back.